src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json

```diff
@@ -564,7 +565,14 @@
       "Build Administrators",
       "Release Administrators"
     ],
-    "AdminInactivityThresholdInDays": 90
+    "AdminInactivityThresholdInDays": 90,
+    "RequiredProperties":[
```

Comment:
RequiredProperties is not required in PS scanner

Reply:
resolved